roles/kubernetes-apps/csi_driver/vsphere/defaults/main.yml

```diff
@@ -31,3 +31,7 @@ vsphere_csi_node_affinity: {}
 # may contain some private data, so it is recommended to set it to false
 # in the production environment.
 unsafe_show_logs: false
+
+# https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/docs/book/features/volume_snapshot.md#how-to-enable-volume-snapshot--restore-feature-in-vsphere-csi-
+# according to the above link , we can controler the block-volume-snapshot parameter
+block_volume_snapshot: false
```

Comment:
Could you add the prefix `vsphere_csi_` to this new parameter?

Reply:
thanks for pr , i have update it again